server.js

```diff
@@ -57,33 +62,70 @@ server.route({
-                // console.log(parsedBodyCommit)
+
                 var commitMessage = parsedBodyCommit.commit.message.replace(/\n/g, '<br>');
-                // console.log("THIS IS THE MESSAGE:", commitMessage)
+                // console.log("This is commit message: " + commitMessage)
 
                 var fileName = parsedBodyCommit.files[0].filename;
                 var patchDiff = parsedBodyCommit.files[0].patch.replace(/\n/g, '<br>');
                 var blobURL = parsedBodyCommit.files[0].blob_url;
-                // console.log(patchDiff)
-                var commitInfo = {
-                    "fileName": fileName,
-                    "blobURL": blobURL,
-                    "commitMessage": commitMessage,
-                    "patchDiff": patchDiff
-                }
 
-                reply(commitInfo)
+
+                // var commitInfo = {
+                //     "fileName": fileName,
+                //     "blobURL": blobURL,
+                //     "commitMessage": commitMessage,
+                //     "patchDiff": patchDiff
+                // }
+
+                var issueBody = Markdown.toHTML("** 🆕🐥☝ First Timers Only. **\n\n This issue is reserved for people who never contributed to Open Source before. We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝 [About First Timers Only](http://www.firsttimersonly.com/).\n\n **🤔 What you will need to know.**\n\n Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.\n\n **:clipboard: Step by Step**\n\n - [x] 🙋 **Claim this issue**: Comment below\n Once claimed we add you as contributor to this repository.\n - [x] 👌 **Accept our invitation** to this repository. Once accepted, assign yourself to this repository\n\n - [x] 👓 **Please review** our [Code of Conduct](http://hood.ie/code-of-conduct/)\n\n In a nutshell: be patient and actively kind\n\n - [x] 🔄 **replace** the `up for grabs` label with `in progress`.\n\n - [x] 📝 **Update** the file["+ fileName +"](" + blobURL + ") in the `hoodie` repository (press the little pen Icon) and edit the line as shown below.\n\n```js \n\ndiff\n\n"+patchDiff+"\n\n ```\n\n - [x] 💾 **Commit** your changes\n\n - [x] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:\n\n1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.\n\n 2. You can [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/)\n\n - [x] 🏁 **Done** :+1: Replace the `in progress` label with `ready`. Ask in comments for a review :) \n\n### 🤔❓ Questions\n\nYou can ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/)");
```

Comment:
you don’t have to turn the Markdown to HTML any more, you can use the Markdown directly for the GitHub issue, it will render it correctly. Maybe that’ll help already?